src/agora/doprecode.cpp

```diff
@@ -63,23 +62,6 @@ Event_data DoPrecode::launch(size_t tag)
         = cfg->get_total_data_symbol_idx_dl(frame_id, symbol_idx_dl);
     const size_t frame_slot = frame_id % kFrameWnd;
 
-    // Mark pilot subcarriers in this block
-    // In downlink pilot symbols, all subcarriers are used as pilots
-    // In downlink data symbols, pilot subcarriers are every
-    // OFDM_PILOT_SPACING subcarriers
-    if (symbol_idx_dl < cfg->DL_PILOT_SYMS) {
-        memset(pilot_sc_flags, 1, cfg->demul_block_size * sizeof(size_t));
-    } else {
-        // Find subcarriers used as pilot in this block
-        memset(pilot_sc_flags, 0, cfg->demul_block_size * sizeof(size_t));
-        size_t remainder = base_sc_id % cfg->OFDM_PILOT_SPACING;
-        size_t first_pilot_sc
-            = remainder > 0 ? (cfg->OFDM_PILOT_SPACING - remainder) : 0;
-        for (size_t i = first_pilot_sc; i < cfg->demul_block_size;
-             i += cfg->OFDM_PILOT_SPACING)
-            pilot_sc_flags[i] = 1;
-    }
-
```

Comment:
Can we keep this part of code as comments for now instead of deleting it? I will debug it later.

Reply:
I brought the code as comment.